pages/data-design/start-using-inspector.mdx

```diff
@@ -25,6 +25,12 @@ Inspector gives you:
 - Implementation status for the events in your entire tracking plan, so you can see which events are actually implemented and whether they’re sent correctly or not.
 - Inspector helps you migrate your tracking plan to Avo. Skip importing things you’re skeptical of. Then leverage Inspector to detect how events and properties are actually tracked, and gradually add them to your tracking plan via Avo branches.
 
+### How does Inspector work
+
+Inspector API extracts the schemas (aka shapes) of the events you trigger (no PII data,
+only event names, property names, and types of properties) and sends those schemas to
+the Avo servers. To repeat: None of the PII data ever flows through Avo servers; only event schemas.
```

Comment:
Removing API here to prevent the misunderstanding of the actual data hitting the API

```suggestion
Inspector API extracts the schemas (aka shapes/metadata) of the events you trigger. Inspector does not process property values or PII data, only event names, property names, and types of properties, and sends those schemas to the Avo servers.

TL;DR: No PII data ever flows through Avo servers; only event schemas.
```